js/ccf-app/test/polyfill.test.ts

```diff
@@ -569,4 +571,25 @@ describe("polyfill", function () {
       assert.equal(foo.get(key_buf), undefined);
     });
   });
+  describe("isValidX509CertChain", function (this) {
```

Comment:
Is this moved down because of the NodeJS crypto bug that you found? If so, it may be worth adding a comment here.